pytensor/tensor/optimize.py

```diff
@@ -378,6 +375,8 @@ def __init__(
         self._fn_wrapped = None
 
     def perform(self, node, inputs, outputs):
+        from scipy.optimize import minimize_scalar as scipy_minimize_scalar
```

Comment:
I suggest the global import strategy used in stats, importing has some non negligible overhead even if a file is already imported.

Reply:
How does it look now

Reply:
good